src/integrations/Axeptio/loader.js

```diff
@@ -0,0 +1,18 @@
+import { LOAD_ORIGIN } from '../../utils/ScriptLoader';
+
+/* eslint-disable */
+// START-NO-SONAR-SCAN
```

Comment:
We need to rename these files to `nativeSdkLoader.js`.
Then, these guarding comments to disable scans should be removed.
See this PR for reference, https://github.com/rudderlabs/rudder-sdk-js/commit/caa2d34fbf4f6619c2cfdc09b267b928d5a1c7f9.